design-assets/README.md

```diff
@@ -0,0 +1,17 @@
+# CMSDS Design Assets
+
+In this folder you will find the following
+
+- CMSDS Sketch file
+- CMSDS fonts (Bitter, Icons, and Open Sans)
+
+To update the Sketch library version create a pull request with the following: 
```

Comment:
What do you think about adding some sections to the README? 
I was thinking it would be nice to have a `## Usage` section that explains what a sketch library is, how to use it via the link, and how it's automatically updated. 

I think the steps below on updating the sketch library could be under a `## Sketch Library Release Process` section or something.

Reply:
That's a good idea - I'm thinking about updating the root `README` so we have all this info in one place as opposed to this separate `README` file.

Reply:
oh, are you saying to combine the `design-assets/README.md` with the root `README`? I think it makes sense to have a root README and separate READMEs for each subsection. To me, the root README should be shorter and contain an overview of the project. Maybe we just need to add links to the sub READMEs in the root README?

Reply:
Updated and ready for another pass @bernardwang